### ISSUE TITLE
Finetuning on LIBERO using pretrained checkpoint with bridgev2-only dataset

### ISSUE THREAD

**Issue by feiyu12138** [OpenDriveLab/UniVLA#64]
Hi Qingwen,

Thank you for clear explanation during previous inquiry, Now i've succesfully reproduced SoTA model reported in the paper. However, when I try to reproduce model pretrained only on Biridgev2(without human data). The performance can't achieve reported result.

My implementation is download released checkpoint which pretrained on bridgev2 and try to finetune it on LIBERO-spatial. I have aligned batch size, learning rate, image processing etc. I use the same setting to train and eval checkpoint pretrained on Bridgev2 and human and reproduce the same performance reported(96.6 vs 96.5). When it comes to Bridge-only checkpoint, i have sucess rate of 91.2(reported result is 95.2). Do you have any suggestion to find potential problem?

Best
